ansible/roles/riakcs/configure/tasks/main.yml

```diff
@@ -58,3 +58,4 @@
 - name: "Start {{ name }}"
   service: name="{{ service }}" state=started enabled=yes
   when: not riakcs_deb_check.get("skipped")
+  tags: after-reboot
```

Comment:
yes, it should start all of riak, stanchion and riak-cs (assuming they're stopped)